src/Nest/Aggregations/Visitor/AggregationVisitor.cs

```diff
@@ -149,6 +151,7 @@ public class AggregationVisitor : IAggregationVisitor
 
 		public virtual void Visit(ITermsAggregation aggregation) { }
 
+		// TODO: make virtual

```

Comment:
TODO?

Reply:
yep, can't change it in 6.x as it would break BWC